sklearn/ensemble/_base.py

```diff
@@ -223,6 +226,15 @@ def _validate_estimators(self):
         # defined by MetaEstimatorMixin
         self._validate_names(names)
 
+        # FIXME: deprecate the usage of None to drop an estimator from the
+        # ensemble
+        if any(est is None for est in estimators):
+            warnings.warn(
+                "Using 'None' to drop an estimator from the ensemble is "
+                "deprecated in 0.22 and support will be dropped in 0.24. "
+                "Use the string 'drop' instead.", DeprecationWarning
```

Comment:
Should we use `FutureWarning` already here or let @NicolasHug 's PR do that?

Reply:
It is not a FutureWarning, isn't it? We want to remove the possibility of doing it.
Regarding the PR of Nicolas, do you have a link to it because I am not which one you are mentioning (getting back from vacation :)).

Reply:
Oh OK, it makes sense now. I see what we speak about :P 
so yes it could be changed in #15080 or here depending what will be included first.